eclair-core/src/main/scala/fr/acinq/eclair/router/Monitoring.scala

```diff
@@ -17,16 +17,40 @@
 package fr.acinq.eclair.router
 
 import fr.acinq.eclair.router.Router.GossipDecision
+import fr.acinq.eclair.wire.ChannelUpdate
 import fr.acinq.eclair.{LongToBtcAmount, MilliSatoshi, getSimpleClassName}
 import kamon.Kamon
-import kamon.metric.Counter
+import kamon.metric.{Counter, MeasurementUnit}
 
 object Monitoring {
 
   object Metrics {
     val FindRouteDuration = Kamon.timer("router.find-route.duration", "Path-finding duration")
     val RouteLength = Kamon.histogram("router.find-route.length", "Path-finding result length")
 
+    val QueryChannelRangeBlocksCount = Kamon.histogram("router.gossip.query-channel-range.blocks-count", "Number of blocks requested in query-channel-range")
+    val QueryChannelRangeRepliesCount = Kamon.histogram("router.gossip.query-channel-range.replies-count", "Number of reply-channel-range replies sent")
+
+    val ReplyChannelRangeIdsCount = Kamon.histogram("router.gossip.reply-channel-range.ids-count", "Number of short channel ids in reply-channel-range")
+    val ReplyChannelRangeBlocksCount = Kamon.histogram("router.gossip.reply-channel-range.blocks-count", "Number of blocks in reply-channel-range")
+    val NewChannelAnnouncements = Kamon.histogram("router.gossip.reply-channel-range.new-channel-announcements", "Number of new channel announcements discovered in reply-channel-range")
```

Comment:
The naming is a bit too generic, shouldn't that be:
```suggestion
    val ReplyChannelRangeNewChannelAnnouncements = Kamon.histogram("router.gossip.reply-channel-range.new-channel-announcements", "Number of new channel announcements discovered in reply-channel-range")
```

Reply:
Maybe we could use `object` instead of very long names? E.g. `ReplyChannelRange.NewChannelAnnouncements`.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1372/commits/3e5a7b082198cc817452737f29a0662e1499e571